src/components/PrivateChat/PrivateChat.tsx

```diff
@@ -280,6 +280,8 @@ class PrivateChat {
         }
     }
     updateModeratorBanner() {
+        if (!this.banner) { return; }
```

Comment:
As mentioned in the other PR, not having this will make it explode, best to fix it somewhere else I suspect?

Reply:
Ah I see what you are asking now.  You are reading this the wrong way around: this is being added :)

Reply:
lol.. teach me to review PR's first thing in the morning. Yes, this is good then!